scripts/trello-to-deck

```diff
@@ -31,7 +31,7 @@ MAX_TITLE_LENGTH = 100
 def truncate(message, length):
     return message if len(message) <= length else message[:length - 1] + "…"
 
-with open(args.input_json, "r") as json_file:
+with open(args.input_json, "r", encoding='UTF-8') as json_file:
```

Comment:
```suggestion
with open(args.input_json, "r", encoding='utf-8') as json_file:
```

Reply:
I think this name is more standard conform: https://docs.python.org/3/library/codecs.html#text-encodings